server/src/api/v1/users/auth-user.ts

```diff
@@ -6,16 +6,20 @@ import jwt from "jsonwebtoken";
 import { Vars } from "../../../vars";
 
 export async function loginUser(req: Request, res: Response) {
+
+    const bcrypt = require ('bcryptjs');
+
     const incomingData: IncomingUser = req.body;
     let success = true;
     let calculatedExpiresIn =  60*60; //expiration after 1h
 
+
     const user = await User.findOne(
         {
             attributes: ['id', 'email', 'is_admin'],
             where: {
                 email: incomingData.email,
-                password: incomingData.password
+                //password: incomingData.password
```

Comment:
Rückgängig machen, sende einfach das gehashte passwort in die Datenbankabfrage, wenn es stimmt, dann kannst du das auf db Ebene machen und musst nichts mehr ändern